packages/rum_sdk/lib/rum_flutter.dart

```diff
@@ -39,7 +38,7 @@ class RumFlutter {
         "device_version": Platform.operatingSystemVersion,
         "dart_version": Platform.version
       }),
-      sdk: Sdk("rum-flutter", "0.0.1", []),
+      sdk: Sdk("rum-flutter", "1.3.5", []),
```

Comment:
I had so add this. Otherwise I got an error when trying to send data to Faro in Grafana Cloud saying that the lowest allowed version was 1.3.5